index.rst

```diff
@@ -922,38 +922,20 @@ dataset is “the persisted form of an in-memory object. It could be a single
 item, a composite, or a collection”). The relationship can occur as many
 datasets in one file, and can also occur as one dataset spread across many files.
 
-Many datasets in one file
-^^^^^^^^^^^^^^^^^^^^^^^^^
+Many (even potentially overlapping) datasets in one file is referred to as a "collection".
```

Comment:
or "A single file containing many (potentially overlapping) datasets is referred to as a 'collection'."